### ISSUE TITLE
Issue with cookbook/choose_protocol.ipynb

### ISSUE THREAD

**Issue by mikemhenry** [OpenFreeEnergy/ExampleNotebooks#138]
![image](https://github.com/OpenFreeEnergy/ExampleNotebooks/assets/11765982/aa7359e9-24ee-4296-a400-68c027d1876b)

I am not sure of we need to define an `__eq__` or something to make the assert more helpful, but right now it is hard to tell what is wrong (or even if there is a difference)

I could write a bit of python to loop through both objects, but for the sake of getting our new showcase out the door, I will just comment out this cell.

**Comment by richardjgowers** [comment]
the eq operator should be working fine, but it won't give you details on why.  This was caused by us updating the default openff FF version, I've fixed this in #99

**Comment by mikemhenry** [comment]
https://github.com/OpenFreeEnergy/ExampleNotebooks/pull/99#issuecomment-2090879201

tl;dr there was an issue with the FF version, but also with things like `25` != `25.0`

**Comment by mikemhenry** [comment]
This PR https://github.com/OpenFreeEnergy/ExampleNotebooks/pull/156 will re-enable the assertion check (which is handy to check if our default settings change)
